elastic/fleet-server
# Log redacted config on config changes

## What is the problem this PR solves?

Debugging the fleet-server can be difficult as it is not clear what config it is running.

## How does this PR solve the problem?

Output redacted configuration when an updated config is received.

## How to test this PR locally

1. Compile a snapshot version of elastic-agent and fleet-server.
2. unzip elastic-agent snapshot.
3. Copy the fleet-server artifact and sha files to the elastic-agent `downloads` dir.
4. Install elastic-agent with a policy that only contains the `fleet-server` integration
5. Review logs to see redacted log config.

## Checklist



- [] ~~I have commented my code, particularly in hard-to-understand areas~~
- [] ~~I have made corresponding changes to the documentation~~
- [] ~~I have made corresponding change to the default configuration files~~

## Related issues

- Closes #1626
